pyigm/guis/igmguesses.py

```diff
@@ -1208,7 +1220,8 @@ def on_draw(self, replot=True, in_wrest=None, rescale=True, fig_clear=False):
                 # Velocity
                 wvobs = (1 + self.z) * wrest
                 wvmnx = wvobs * (1 + np.array(self.psdict['x_minmax']) / c_kms)
-                velo = (self.spec.wavelength/wvobs - 1.) * c_kms * u.km/u.s
+                # velo = (self.spec.wavelength/wvobs - 1.) * c_kms * u.km/u.s
+                velo = ltu.give_dv(self.spec.wavelength/wrest - 1., self.z)
```

Comment:
Yeah, we should definitely avoid having both of these sets of methods.

Please do go for a refactor.  I suspect v_from_z is used in quite a few places
so probably best to edit that one.  Will raise as an Issue in linetools